src/__generated__/app.graphql

```diff
@@ -0,0 +1,607 @@
+"""
```

Comment:
hmm I think this is defaulting to `example app`... I dunno if we want this schema since this won't be peoples apps

Reply:
Not sure I'm understanding, can you explain this one a bit more?